docs/docs/analytics.mdx

```diff
@@ -0,0 +1,126 @@
+---
+id: analytics
+sidebar_label: Analytics
+title: Analytics
+description:
+abstract: Rasa Pro Analytics is an ETL pipeline that extracts model, training, and conversation data from the Rasa platform and creates a queryable and extensible database schema.
```

Comment:
In terms of structure, we might need a section where we break down what happens at every ETL stage, so that customers who want to plug-in at a certain stage know how the data is structured at that stage.

Reply:
I think we can't allow customizations at this point (and I think we shouldn't). We rather directly want to get the feedback of what is missing or what they'd like to do and improve the pipeline based on that feedback and properly plan extension points. At this time, they can combine the data with other data in their AWS Redshift data warehouse.